cmd/balance-agent/main.go

```diff
@@ -303,6 +306,10 @@ func getApp() *cli.App {
 			Usage:  "Ignore CLN socket",
 			Hidden: true,
 		},
+		&cli.StringFlag{
```

Comment:
I would be nice if we would support yaml config as well instead of so many flags =)

Reply:
Agree, but ideally you don't need to set anything and it just works.